.github/component_owners.yml

```diff
@@ -0,0 +1,24 @@
+# Keep all in alphabetical order
+components:
```

Comment:
I got on these ownerships by looking through the git history (additions of the component and significant changes, reviews).